### PR TITLE
Create ip_spoofing_via_http_headers.md

### PR DESCRIPTION
**Pull request by Ahmad-Reza-UT**

### CODE REVIEW

**Comment by kingthorin** [comment]
Why do you keep creating new PRs? You already have two open for this content.

**Comment by Ahmad-Reza-UT** [comment]
I am sorry
This is my first time so I am not well aware about it.

But I edited all the problems

**Comment by Ahmad-Reza-UT** [comment]
> Why do you keep creating new PRs? You already have two open for this content.

I am sorry I wasn't aware about it and This is my first time but I have edited ethe issues yiu mentioned before

File: pages/attacks/ip_spoofing_via_http_headers.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+## Attack Scenario
+
+### Workflow
+
+**Introduction to IP Spoofing:**
+Internet Protocol (IP) spoofing is a type of malicious attack where the threat actor hides the true source of IP packets to make it difficult to know where they came from. The attacker creates packets, changing the source IP address to impersonate a different computer system, disguise the sender's identity or both. The spoofed packet's header field for the source IP address contains an address that is different from the actual source IP address. IP spoofing is a technique often used by attackers to launch distributed denial of service (DDoS) attacks and man-in-the-middle attacks against targeted devices or the surrounding infrastructures. The goal of DDoS attacks is to overwhelm a target with traffic while hiding the identity of the malicious source, preventing mitigation efforts.
```
Comment (kingthorin) [inline pages/attacks/ip_spoofing_via_http_headers.md:22]: This has zero to do with headers

File: pages/attacks/ip_spoofing_via_http_headers.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+# IP Spoofing via HTTP Headers
+
+## Description
+
+In the realm of web security, understanding the nuances of IP spoofing becomes paramount as malicious actors exploit vulnerabilities in HTTP headers to deceive systems. This article delves into the intricacies of IP spoofing, focusing on the utilization of specific HTTP headers as an attack vector. By examining the risks associated with trusting headers like X-Forwarded-For, we unveil the potential consequences and offer insights into securing applications against these threats.
+
+## Attack Scenario
+
+### Workflow
```
Comment (kingthorin) [inline pages/attacks/ip_spoofing_via_http_headers.md:19]: This doesn't outline a workflow

File: pages/attacks/ip_spoofing_via_http_headers.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+2. **Rate Limits Based on IP Addresses:**
+   - *Scenario:* A login system allows a limited number of failed login attempts from each unique IP address within a specified timeframe.
+   - *Implementation:* The application tracks failed login attempts along with the corresponding IP addresses. If the allowed limit is exceeded, further login attempts from that IP are temporarily restricted.
+
+
+
+
+### Examples Related to IP Spoofing
+
+1. ### DDoS Attacks
```
Comment (kingthorin) [inline pages/attacks/ip_spoofing_via_http_headers.md:46]: This has nothing to do with headers

**Review by kingthorin — CHANGES_REQUESTED** [review]
Please do not open another PR simply re-edit the content associated with this one.

**Comment by Ahmad-Reza-UT** [comment]
How long does it take to review my article?

**Comment by Ahmad-Reza-UT** [comment]
I have edited all the things you mentioned

**Comment by kingthorin** [comment]
Thanks. This content is handled by volunteers. So there's no SLA.

I'll try to have another look at it this evening.

**Review by kingthorin — APPROVED** [review]

**Pull request merged**